### DISCUSSION TITLE
This project has already been connected to Rowy

### DISCUSSION THREAD

**Discussion by Beardmix** [AntlerVC/firetable#1257] [Support / Q&A] [answered]
Hi!

I wanted to connect my firebase project to Rowy, but stopped during the project creation phase.
I tried to restart the creation of my first Rowy project but it complains that I already added the Firebase project.

![Screenshot 2023-05-11 155450](https://github.com/rowyio/rowy/assets/4427443/d1639790-00fd-4d4d-a662-114153085f1d)

I never finished adding it, so I can not delete/unlink it.

What can I do?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @Beardmix, could you share the Project ID and your email id? Also, are you the owner of this project?

**Reply by Beardmix** [reply]
Hi @BeeBombshell, I wrote you a private email with the details for security/privacy reasons.

I am the owner of both the Rowy and Firebase projects.

**Reply by BeeBombshell** [reply]
@Beardmix, did you email support@rowy.io? I'm not sure if I can see it, could you share your email id?

**Reply by BeeBombshell** [reply]
Got it! 👍
